uaclient/contract.py

```diff
@@ -163,21 +125,6 @@ def request_contract_machine_attach(self, contract_token, machine_id=None):
         self.cfg.write_cache('machine-token', machine_token)
         return machine_token
 
-    def request_contract_machine_detach(self, contract_id, user_token):
```

Comment:
I believe we **should** call this and it is an error of omission that we aren't. We should check in UA checkin meeting today that the ua-contracts service expects this call to close the loop for accounting on active 'attached' machines.

Reply:
@blackboxsw I've dropped the commit that removed this method.